### PR TITLE
Larrys branch

### PR DESCRIPTION
**Pull request by LarryHu208**
hi this is my pull request for a working users profile

### CODE REVIEW

File: dmsite/venv/src/ProfileComponent.js [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
-                    </Col>
-                    <Col md="5">
-                        <Results name={"Classifications"} />
-                    </Col>
-                    <Col md="4">
-                        <DisplayColumn name={'Display Actions'} />
-                    </Col>
-                </Row>
+            <Container fluid style={{ justifyContent: 'center' }}>
+              <Card style={{ width: '60%', justifyContent: 'center', alignSelf: "center" }}>
```
Comment (lzcheung) [inline dmsite/venv/src/ProfileComponent.js]: It peeves me that you use single quotes for the first 2 values and double quotes in the alignSelf. Try to keep things consistent.

File: dmsite/venv/src/ProfileComponent.js [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
+                  <InputGroup>
+                    <InputGroupAddon addonType="prepend">
+                      <InputGroupText>@Security</InputGroupText>
+                    </InputGroupAddon>
+                    <Input />
+                  </InputGroup>
+                  <br />
+                </CardBody>
+                <div className="d-flex justify-content-center pt-2">
+                  <Button color="primary" size="md" className="mr-1 btn-block mt-0" onClick={() => { this.props.history.push('/home'); }} >Save</Button>
```
Comment (lzcheung) [inline dmsite/venv/src/ProfileComponent.js]: I'm guessing these buttons don't arent finished with their functionality?

Comment (lzcheung) [reply dmsite/venv/src/ProfileComponent.js]: Should add 2 function that handles the saving and going back respectively. You can add the navigation inside those functions and leave a TODO comment if you plan on adding the functionality later.

File: dmsite/venv/src/ProfileComponent.js [hunk 3]
```diff
@@ -1,29 +1,52 @@
...
+                  <Button color="primary"size="md" className="btn-block mt-0" onClick={() => { this.props.history.push('/home'); }} >Go Back</Button>
```
Comment (lzcheung) [inline dmsite/venv/src/ProfileComponent.js]: Change the text on this to be Back or Cancel. Much more standard. Also differentiate the button colors of save and back. I would try to disable the save button until text is entered in the inputs.

**Comment by stbradle** [comment]
I agree with Liang on every point he made. Also, make sure your branch is up to date with master.

File: dmsite/venv/src/HomeComponent.js [hunk 4]
```diff
@@ -9,7 +9,11 @@ import Results from './ClassificationColumn/Results';
 class Home extends Component{
     render(){
         return (
-            <Container fluid>
+            <Container fluid styles = {{
+                                                                          backgroundImage: 'url(require("images/DND.jpg"))',
+                                                                          overflow: 'hidden',
+                                                                      }}>
```
Comment (gdaswegan) [inline dmsite/venv/src/HomeComponent.js]: Please fix your tabs here - its very jarring to read this.

**Comment by gdaswegan** [comment]
Pretty much just echo Liang's thoughts.

**Comment by gdaswegan** [comment]
@LarryHu208 As a reminder, this needs to be changed, and you need to resolve merge conflicts before we can pull this into master. Please do so, so that we have a solid working version of Master tomorrow.

**Comment by LarryHu208** [comment]
This is the formatting changes. I haven't created any additional add-ons. Im on that right now!

**Comment by cedaley** [comment]
When you commit your changes are you committing things you didn't work on?

**Comment by stbradle** [comment]
When you commit things, please make the commit message something thats at least somewhat descriptive.

File: dmsite/venv/src/Authentication/Login.js [hunk 5]
```diff
@@ -16,7 +16,7 @@ class Login extends Component {
   }
 
   uploadPressed() {
-    const path = '/home';
+    const path = '/campaign';
```
Comment (gdaswegan) [inline dmsite/venv/src/Authentication/Login.js]: Campaign as a page doesn't exist - probably should be left as home.

File: dmsite/venv/src/Profile/ProfileComponent.jsx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+              <br />
+              <InputGroup>
+                <InputGroupAddon addonType="prepend">
+                  <Button id="PopoverFocus" type="button">@Password</Button>
+                  <Popover trigger="focus" placement="bottom" target="PopoverFocus">
+                    <PopoverHeader>Focus Trigger</PopoverHeader>
+                    <PopoverBody>hi</PopoverBody>
+                  </Popover>
+                </InputGroupAddon>
+                <Input placeholder="Password123" />
```
Comment (gdaswegan) [inline dmsite/venv/src/Profile/ProfileComponent.jsx]: Probably should be left as regular placeholders; same with UsernameMan above

File: dmsite/venv/src/home/ClassificationColumn/Results.jsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import 'filepond/dist/filepond.min.css';
+import {
+  Input, Card, Button, CardTitle, CardBody, ListGroup, ListGroupItem,
+} from 'reactstrap';
+
+const parseData = (data) => {
+  const finalData = data.map((obj, index) => {
+    console.log(obj);
```
Comment (lzcheung) [inline dmsite/venv/src/home/ClassificationColumn/Results.jsx]: Remove console.log

File: dmsite/venv/src/Profile/ProfileComponent.jsx [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      popoverOpen: false,
+    };
+  }
+
+  toggle() {
```
Comment (cedaley) [inline dmsite/venv/src/Profile/ProfileComponent.jsx]: Add comments if toggle feature isn't working

**Pull request merged**